Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM julia:1.3
+
+WORKDIR /home
+
+COPY test/REQUIRE /home
+
+RUN for dep in $(cat REQUIRE); do julia -e "using Pkg; Pkg.add(\"${dep}\")"; done
```

Comment:
To replace this use of REQUIRE, I think you could use something like (untested!)

```Docker
COPY test/Project.toml /home
RUN julia --project=/home "using Pkg; Pkg.instantiate();"
```

(More generally I'm not sure what this Docker file is for, but I assume it's important to you :-) )

Reply:
(The dockerfile was very useful for running the tests in different Julia versions without having to compile and install them all :D).